### PR TITLE
Updating legal documents

### PR DESCRIPTION
**Pull request by waltfy**
📜 **LEGAL DOCUMENTS** 📜 

This change includes:

- Renaming `'Customer agreement'` to `'Website terms of use'`
- Updating the `'Website terms of use'` document
- Updating the `'Privacy Policy'` document

### CODE REVIEW

**Comment by clippy** [comment]
**I have manually checked the visuals in:**

- [ ] Chrome
- [ ] IE11
- [ ] Firefox
- [ ] Safari
- [ ] iPad

**Comment by waltfy** [comment]
@harrison review? We're currently waiting on some wording changes from Amed — should still be fine to review though.

File: app/pages/legal/privacy/legal-privacy.en.js [hunk 1]
```diff
@@ -51,125 +52,150 @@ export default class LegalPrivacyEn extends React.Component {
...
 
         <h3 className='u-text-heading-light u-color-dark-gray u-margin-Vm u-text-s'>
-          Cookies
+          Disclosure of your information
         </h3>
+
         <p className='para'>
-          Our web site uses cookies – small text files stored on your computer – in two ways.
+          We may disclose your personal information to third parties if we are under a duty to disclose or share
+          your personal data in order to comply with any legal obligation, or in order to enforce or apply our <a href='https://gocardless.com/legal'>Terms and Conditions</a> and/or other agreements entered into with you;
```
Comment (harrison) [inline app/pages/legal/privacy/legal-privacy.en.js]: <Link

File: app/pages/legal/privacy/legal-privacy.es.js [hunk 2]
```diff
@@ -51,125 +52,150 @@ export default class LegalPrivacyEs extends React.Component {
...
 
         <h3 className='u-text-heading-light u-color-dark-gray u-margin-Vm u-text-s'>
-          Cookies
+          Disclosure of your information
         </h3>
+
         <p className='para'>
-          Our web site uses cookies – small text files stored on your computer – in two ways.
+          We may disclose your personal information to third parties if we are under a duty to disclose or share
+          your personal data in order to comply with any legal obligation, or in order to enforce or apply our <a href='https://gocardless.com/legal'>Terms and Conditions</a> and/or other agreements entered into with you;
```
Comment (harrison) [inline app/pages/legal/privacy/legal-privacy.es.js]: <Link

File: app/css/components/list.scss [hunk 3]
```diff
@@ -13,3 +13,35 @@
 .list--numbered {
   list-style: decimal;
 }
+
+.numeric-decimals {
```
Comment (harrison) [inline app/css/components/list.scss]: move to numeric-decimals component?

File: app/css/components/numeric-decimals.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
+.numeric-decimals {
```
Comment (Ken-Lo) [inline app/css/components/numeric-decimals.scss]: @waltfy Is there any reason we don't use the `<ol>` tag?:)

Comment (waltfy) [inline app/css/components/numeric-decimals.scss]: @Ken-Lo no particular reason — we could change it to `ol` :)

**Pull request merged**